Resources/Private/Data/MetadataDefaults.php

```diff
@@ -93,7 +93,7 @@
             ],
         ],
         'default_value' => '',
-        'wrap' => '',
+        'wrap' => "key.wrap = <dt class=\"tx-dlf-metadata-author\">|</dt>\nvalue.required = 1\nvalue.split.token.char = 31\nvalue.split.cObjNum = 1,2\nvalue.split.1 = LOAD_REGISTER\nvalue.split.1.tx_dlf_metadata_author_name.current = 1\nvalue.split.2 = COA\nvalue.split.2.1 = LOAD_REGISTER\nvalue.split.2.1.tx_dlf_metadata_author_uri.current = 1\nvalue.split.2.2 = TEXT\nvalue.split.2.2.value = {register:tx_dlf_metadata_author_name}\nvalue.split.2.2.value.stdWrap.insertData = 1\nvalue.split.2.2.value.stdWrap.typolink.parameter = {register:tx_dlf_metadata_author_uri}\nvalue.split.2.2.value.stdWrap.typolink.parameter.insertData = 1\nvalue.split.2.2.value.stdWrap.typolink.ifNotEmpty = 1\nvalue.wrap = <dd class=\"tx-dlf-metadata-author\">|</dd>",
```

Comment:
As you feard, this TypoScript has no output.

In case of existing valueURIs, the following TypoScript would work. But this still makes no output of authors if valueURI is missing.
```
key.wrap = <dt class="tx-dlf-metadata-author">|</dt>
value.required = 1
value.split.token.char = 31
# use OptionSplit to handle resulting array separately
value.split.cObjNum = 1 || 2
# load register in cObject
value.split.1.1 = LOAD_REGISTER
value.split.1.1.tx_dlf_metadata_author_name.current = 1
value.split.2 = COA
value.split.2.1 = LOAD_REGISTER
value.split.2.1.tx_dlf_metadata_author_uri.current = 1
value.split.2.2 = TEXT
value.split.2.2.value = {register:tx_dlf_metadata_author_name}
# insertData of course in value not in stdWrap
value.split.2.2.value.insertData = 1
value.split.2.2.value.stdWrap.typolink.parameter = {register:tx_dlf_metadata_author_uri}
value.split.2.2.value.stdWrap.typolink.parameter.insertData = 1
value.split.2.2.value.stdWrap.typolink.ifNotEmpty = 1
value.wrap = <dd class="tx-dlf-metadata-author">|</dd>
```

The following code would work in both usecases: authors with valueURI and without. As usual with TypoScript: there might be a smarter solution. But I stop looking for it now ;-)

```
key.wrap = <dt class="tx-dlf-metadata-author">|</dt>
value.required = 1
value.split {
	token.char = 31
	cObjNum = 1
	1 = COA
	1 {
		20 = CASE
		20 {
                        # register SPLIT_COUNT is 0 on first split component (name) and 1 on second (URI)
			key.data = register:SPLIT_COUNT

                        # no not output the split component but save in  a register
			0 = COA
			0 {
				10 = LOAD_REGISTER
				10.tx_dlf_metadata_author_name.current = 1
			}

			1 = COA
			1 {
				10 = LOAD_REGISTER
				10.tx_dlf_metadata_author_uri.current = 1			
			}
			
			
		}
	}
}
# append a cObject and use the one or two registers from above
value.postCObject = TEXT
value.postCObject {
	value = {register:tx_dlf_metadata_author_name}
	value.insertData = 1
	value.stdWrap.typolink.parameter = {register:tx_dlf_metadata_author_uri} _blank external
	value.stdWrap.typolink.title = {register:tx_dlf_metadata_author_name}
	value.stdWrap.typolink.parameter.insertData = 1
	value.stdWrap.typolink.ifNotEmpty = 1
}

value.wrap = <dd class="tx-dlf-metadata-author">|</dd>
```

Reply:
Is there are reason for all the `COA` cObjects? I removed them and now the typoscript looks like this:
```
key.wrap = <dt class=\"tx-dlf-metadata-author\">|</dt>
value.required = 1
value.split.token.char = 31
value.split.cObjNum = 1
value.split.1 = CASE
value.split.1.key.data = register:SPLIT_COUNT
value.split.1.0 = LOAD_REGISTER
value.split.1.0.tx_dlf_metadata_author_name.current = 1
value.split.1.1 = LOAD_REGISTER
value.split.1.1.tx_dlf_metadata_author_uri.current = 1
value.postCObject = TEXT
value.postCObject.value = {register:tx_dlf_metadata_author_name}
value.postCObject.value.insertData = 1
value.postCObject.value.stdWrap.typolink.parameter = {register:tx_dlf_metadata_author_uri} _blank external
value.postCObject.value.stdWrap.typolink.parameter.insertData = 1
value.postCObject.value.stdWrap.typolink.title = {register:tx_dlf_metadata_author_name}
value.postCObject.value.stdWrap.typolink.ifNotEmpty = 1
value.wrap = <dd class=\"tx-dlf-metadata-author\">|</dd>
```

Reply:
At least `value.splilt.1` must not be overwritten by `CASE` as it contains the split result.
Moving the `CASE` to `value.split.1.1` does the job:

<pre>
key.wrap = <dt class=\"tx-dlf-metadata-author\">|</dt>
value.required = 1
value.split.token.char = 31
value.split.cObjNum = 1
value.split.1.1 = CASE
value.split.1.1.key.data = register:SPLIT_COUNT
value.split.1.1.0 = LOAD_REGISTER
value.split.1.1.0.tx_dlf_metadata_author_name.current = 1
value.split.1.1.1 = LOAD_REGISTER
value.split.1.1.1.tx_dlf_metadata_author_uri.current = 1
value.postCObject = TEXT
value.postCObject.value = {register:tx_dlf_metadata_author_name}
value.postCObject.value.insertData = 1
value.postCObject.value.stdWrap.typolink.parameter = {register:tx_dlf_metadata_author_uri} _blank external
value.postCObject.value.stdWrap.typolink.parameter.insertData = 1
value.postCObject.value.stdWrap.typolink.title = {register:tx_dlf_metadata_author_name}
value.postCObject.value.stdWrap.typolink.ifNotEmpty = 1
value.wrap = <dd class=\"tx-dlf-metadata-author\">|</dd>
</pre>